modules/Users/Users.php

```diff
@@ -696,7 +721,6 @@ public function fill_in_additional_detail_fields() {
 	 * @returns user info in $this->column_fields array:: Type array
 	 */
 	public function retrieveCurrentUserInfoFromFile($userid) {
-		global $adb, $site_URL;
```

Comment:
recover this code, you are undoing an existing change that was added a few days ago